addons/studio-invite-button/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ addon, global, console }) {
+// RegExp to test for a valid url and get the studio id
+const urlRegex = /^https?:\/\/scratch\.mit\.edu\/studios\/(\d+)\/comments($|\/)/;
+// invites a user to the studio
+function inviteToStudio(user) {
+  return fetch(`https://scratch.mit.edu/site-api/users/curators-in/${urlRegex.exec(window.location.href)[1]}/invite_curator/?usernames=${user}`, {
+    headers: {
+      'x-csrftoken': addon.auth.csrfToken,
+      'x-requested-with': 'XMLHttpRequest',
+      referer: 'https://scratch.mit.edu',
+
+    },
+    body: null,
+    method: 'PUT',
+  });
+}
+// checks if the user is a manager and the URL is correct
+if (document.getElementById('description').classList.contains('editable') && urlRegex.test(window.location.href)) {
+  Array.from(document.getElementsByClassName('comment')).forEach((comment) => {
+    const inviteButton = document.createElement('span'); // create the button
+    inviteButton.innerHTML = 'Invite';
+    inviteButton.classList.add('actions');
+    inviteButton.style.visibility = 'hidden';
+    inviteButton.style.color = 'rgb(157, 157, 157)';
+
+    comment.querySelector('.actions-wrap').appendChild(inviteButton);
+    comment.addEventListener('mouseover', () => {
+      inviteButton.style.visibility = 'visible'; // should only be visible when the mouse is hovering over the comment
+    });
+    comment.addEventListener('mouseout', () => {
+      inviteButton.style.visibility = 'hidden';
+    });
```

Comment:
If you add the "report" class to the inviteButton, you don't need this code.

Reply:
I don't want to risk other side affects that could happen when I do that

Reply:
there are no "side affects"

Reply:
okay